.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,66 +1,26 @@
----
-name: Pull request template
-about: Template for interchain security templates
-title: ''
-labels: ''
-assignees: ''
-
----
-
-
 # Description
 
 Please include a summary of the changes and the related issue. If the issue was ambiguous try to clarify it in this section.
 
-
-# Linked issues
+## Linked issues
 
 Closes: `#<issue>`
 
+## Type of change
 
-# Type of change
-
-Please delete options that are not relevant.
-
-- [ ] Non-breaking changes
-- [ ] New feature (adding functionality)
-- [ ] Breaking change (fix, feature or refactoring that changes existing functionality)
-- [ ] Requires state migrations
-- [ ] Proto files changes
-- [ ] Updates in store keepers or store keys
-- [ ] Changes in genesis (import/export)
-- [ ] Testing
-- [ ] Dependency management (updates dependencies, adds replace calls in go.mod, go mod tidy)
-- [ ] Documentation updates
-
-
-# How was the feature tested?
-
-- [ ] Unit tests
```

Comment:
maybe leave this in? Not sure if people are using it. If not, just delete :D

Reply:
Imo "how was this feature tested" is too broad. GH already shows you below which types of tests are passing or failing. The most important consideration in my mind is: 1. what new or existing tests are acting as regression tests, 2. what new or existing tests are validating new behavior. 

Just my two cents :), not trying to take things away that people like